dev/com.ibm.ws.session.cache/resources/com/ibm/ws/session/store/cache/resources/WASSessionCache.nlsprops

```diff
@@ -19,4 +19,32 @@
 
 INCORRECT_URI_SYNTAX=SESN0300E: Invalid syntax in httpSessionCache URI.  Cause is {0}.
 INCORRECT_URI_SYNTAX.explanation=The syntax of the specified httpSessionCache JCache configuration URI is invalid.
-INCORRECT_URI_SYNTAX.useraction=Correct the syntax of the specified httpSessionCache configuration URI.
\ No newline at end of file
+INCORRECT_URI_SYNTAX.useraction=Correct the syntax of the specified httpSessionCache configuration URI.
+
+ERROR_REMOVING_SESSION=SESN0301E: An error occurred when trying to remove a session from the cache. The exception is: {0}.
+ERROR_REMOVING_SESSION.explanation=An error occurred when trying to remove a session from the cache. See exception for further details.
```

Comment:
(applies to all added messages) The explanation should never restate the error message.  If there is no additional information that would be helpful, just omit that part, or replace with something like, "An operation against the cache failed due to the specified exception."